FFE_AGENT_HIERARCHY_DESIGN.md

```
@@ -0,0 +1,699 @@
+# FFE Sub-Agent Hierarchy Design
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)